tests/PHPStan/Rules/Methods/data/true-typehint.php

```diff
@@ -0,0 +1,25 @@
+<?php // lint >= 8.2
+
+namespace NativeTrueType;
+
+use function PHPStan\Testing\assertType;
+
+class Truthy {
+	public true $truthy = true;
+
+	public function foo(true $v): true {
+		assertType('true', $v);
+	}
+
+	function trueUnion(true|null $trueUnion): void
+	{
+	}
+
+	function trueUnionReturn(): true|null
```

Comment:
How are the other two methods tested? I'd expect more assertType calls here.

Reply:
done